### PR TITLE
Fixed Office endpoint

### LINKED ISSUE #59: Fix Office endpoing

### PR DESCRIPTION
**Pull request by Andrei-Ha**
Fixed #59

### CODE REVIEW

File: src/Api/Controllers/OfficesController.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 namespace Exadel.OfficeBooking.Api.Controllers
 {
     [ApiController]
-    [Route("api/[controller]/[action]")]
+    [Route("api/office")]
```
Comment (khamza85) [inline src/Api/Controllers/OfficesController.cs:12]: `[Route("api/[controller]")]`

File: src/Api/Controllers/OfficesController.cs [hunk 2]
```diff
@@ -18,63 +18,48 @@ public OfficesController(IOfficeService officeService)
...
-            var offices = await _officeService.GetOffices(filterModel);
-
+            var offices = await _officeService.GetOffices();
             return Ok(offices);
         }
 
-        // GET api/<OfficesController>/5
         [HttpGet("{id}")]
-        [Produces("application/json")]
         public async Task<ActionResult<OfficeGetDto>> GetById(Guid id)
```
Comment (khamza85) [inline src/Api/Controllers/OfficesController.cs:29]: Same here, just `Type<OfficeGetDto>` should also be ok

**Review by SandroShera — APPROVED** [review]

File: src/Api/Controllers/OfficesController.cs (hunk 2 above)
Comment (Andrei-Ha) [reply src/Api/Controllers/OfficesController.cs:29]: If I change this, I can't return the response NotFound(new {message = "The requested office was not found"})

**Comment by Andrei-Ha** [comment]
Changed from OfficesController to OfficeController

**Review by khamza85 — APPROVED** [review]

**Review by Bazilis — APPROVED** [review]

**Pull request merged**